.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve slo-exporter
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+Please see the dedicated section of [our contributing guideline](https://github.com/seznam/slo-exporter/blob/master/CONTRIBUTING.md#reporting-bugs).
+
+**Describe the bug**
```

Comment:
I'd add a section about the environment the bug happened in. Version of slo-exporter, virtuaization sytem etc maybe?

Reply:
All this is specified in the referenced document - https://github.com/seznam/slo-exporter/blob/master/CONTRIBUTING.md#reporting-bugs. 

Let's either remove it from the document and specify it here, or leave it as it is now.

Reply:
I'd say the template is enough, and we do not need to reference the contribution guide at all. 
Let's make it easier and complicate it once there is a need for it?

Reply:
https://github.com/seznam/slo-exporter/pull/47/commits/173c9b63a334c1b565df34891349bd0ab91f901d